<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sampler</class>
 <widget class="QWidget" name="Sampler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>75</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="textureName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="placeholderText">
      <string>None Selected</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btnDelTextureMap">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string>Reset idle track to nothing</string>
       </property>
       <property name="text">
        <string>,..</string>
       </property>
       <property name="icon">
        <iconset resource="../resource/resource.qrc">
         <normaloff>:/16x16_ico_png/reset.png</normaloff>:/16x16_ico_png/reset.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnAddTextureMap">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../resource/resource.qrc">
         <normaloff>:/16x16_ico_png/add.png</normaloff>:/16x16_ico_png/add.png</iconset>
       </property>
       <property name="popupMode">
        <enum>QToolButton::MenuButtonPopup</enum>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonFollowStyle</enum>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
       <property name="arrowType">
        <enum>Qt::NoArrow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="spriteFps">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="specialValueText">
      <string/>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
     </property>
     <property name="value">
      <double>0.200000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lblFps">
     <property name="text">
      <string>FPS</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
